src/poliastro/tests/tests_threebody/test_soi.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+from astropy import units as u
+from astropy.tests.helper import assert_quantity_allclose
+
+from poliastro.bodies import (
+    Body,
+    Earth,
+    Jupiter,
+    Mars,
+    Mercury,
+    Moon,
+    Neptune,
+    Pluto,
+    Saturn,
+    Sun,
+    Uranus,
+    Venus,
+)
+from poliastro.threebody.soi import laplace_radius, hill_radius
+
+
+def test_laplace_radius():
+    # Data from Table A.2., Curtis "Orbital Mechanics for Engineering Students"
+    data = [
+        # body, SOI radius (m)
+        (Sun, None),
+        (Mercury, 1.12e8),
+        (Venus, 6.16e8),
+        (Earth, 9.25e8),
+        # (Moon, 6.61e7),
+        (Mars, 5.77e8),
+        (Jupiter, 4.82e10),
+        (Saturn, 5.48e10),
+        (Uranus, 5.18e10),
+        (Neptune, 8.66e10),
+        # (Pluto, 3.08e9)
+    ]
+    for row in data:
+
+        body, expected_r_SOI = row
+        if expected_r_SOI is not None:
```

Comment:
While we are at it, can we just include the units in the tuple and use `pytest.mark.parametrize`?

Reply:
Sure. Will do that!